src/pages/RoomMembersPage.js

```diff
@@ -250,7 +240,7 @@ function RoomMembersPage(props) {
             testID={RoomMembersPage.displayName}
         >
             <FullPageNotFoundView
-                shouldShow={_.isEmpty(props.report) || !isPolicyMember}
+                shouldShow={_.isEmpty(props.report) || (ReportUtils.isPublicRoom(props.report) && !isPolicyMember)}
```

Comment:
What happens if the user is accessing a deep link to a room in which he is neither a member nor part of the workspace?

Reply:
I believe this is handled by `withReportOrNotFound`

Reply:
I believe these cases are handled already by withReportOrNotFound